src/main/java/org/folio/dew/controller/BulkEditController.java

```diff
@@ -334,6 +342,16 @@ public ResponseEntity<String> uploadCsvFile(UUID jobId, MultipartFile file) {
     }
   }
 
+  private String saveTemporaryIdentifiersFile(UUID jobId, MultipartFile file) throws IOException {
+    var tempDir = System.getProperty(TMP_DIR_PROPERTY) + PATH_SEPARATOR + springApplicationName + PATH_SEPARATOR + jobId;
```

Comment:
Plz, use Path options to create path for directory, path separator in that case will be operational system independent.